### DISCUSSION TITLE
配置不能正确初始化

### DISCUSSION THREAD

**Discussion by taynpg** [Martins3/My-Linux-config#153] [Q&A] [answered]
![图片](https://github.com/Martins3/My-Linux-Config/assets/59216127/b8eeccec-3a2e-409f-ac65-ccc0e52cc3da)

大佬，请教个问题，我按照你的教程clone项目后，nvim启动会图中的错误，不清楚什么情况，请求帮助。
nvim版本：0.9.4
系统：Debian 12.2
rsync这个命令我是安装的了。

**Comment by Martins3** [comment]
尝试 `~/.local/share/nvim/lazy/rsync.nvim && make -j8`

这个插件搞了好一个问题，我看看到时候有什么好办法解决下

**Reply by taynpg** [reply]
试了一下，倒是没有报错了，但是直接闪退了，就是进去之后，自己自动退回bash了。

**Reply by Martins3** [reply]
试试 rm -r ~/.local/share/nvim/sessions

**Reply by taynpg** [reply]
![图片](https://github.com/Martins3/My-Linux-Config/assets/59216127/5783378f-b042-41e6-a6c5-7dd506528867)

nil 组件构建失败。

**Reply by Martins3** [reply, marked as answer]
https://github.com/Martins3/My-Linux-Config/blob/5f5f575e532290dffdd673571c3ed5dcbc92b74d/nvim/lua/usr/lsp/mason.lua#L41 中将`, "nil_ls"` 删除掉

**Reply by taynpg** [reply]
不知道是不是我系统问题，还是闪退什么的，先这样吧，非常感谢大佬的帮助。

**Reply by Martins3** [reply]
`dmesg` 看看，是不是直接 crash 了，还是因为 session 的原因

**Reply by taynpg** [reply]
![image](https://github.com/Martins3/My-Linux-Config/assets/59216127/c1670bf3-06cd-408a-b537-9f1e7d7f4e1d)

这些组件安装不成功，还有就是我使用nvim + dir的形式会退出来，是不是配置不支持这种用法，直接nvim能进去，但是这些组件不成功。
![image](https://github.com/Martins3/My-Linux-Config/assets/59216127/8a0c8821-b82d-4850-8b18-0cec6587fe4f)

**Reply by Martins3** [reply]
设置终端的代理，打开 nvim ，执行 `:Mason` ，然后等待，最后看看啥结果

**Reply by taynpg** [reply]
好像找到问题了，nodejs版本太低？还有就是npm没装。nodejs 12.几的。
